src/components/common/ColorPicker.js

```diff
@@ -0,0 +1,40 @@
+import React, { useRef } from 'react';
+import { Overlay, Popover } from 'react-bootstrap';
+import { BlockPicker } from 'react-color';
+import '../../styles/colorpicker.css';
+
+const ColorPicker = props => {
+  const { children, show, color, onColorChange } = props;
+
+  const defaultColors = ['#ededed', '#7189bf', '#df7599', '#ffc785', '#72d6c9'];
```

Comment:
How about implementing a bit richer color palet?
![image](https://user-images.githubusercontent.com/51719931/79473273-31633b00-8005-11ea-9419-a79263f5945e.png)
```suggestion
    const defaultColors = [
    '#ffef7f',
    '#ffc785',
    '#f1daf5',
    '#df7599',
    '#ba0120',
    '#7189bf',
    '#72d6c9',
    '#77da57',
    '#ededed',
    '#22222a'
  ];
```